### PR TITLE
PXD-1221 Hooked up filters, table, charts to Arranger

### PR DESCRIPTION
**Pull request by abgeorge7**
- Created Arranger wrapper that exposes the Arranger data to our components
- Pulled Arranger data into charts
- Modified DataExplorer component to lay out components as in spec
- Added some instructions to the ReadME about Arranger/Storybook
Note: to see these successfully in Storybook you will need to run Arranger/our gen3-arranger server locally.

Next steps: try to use the custom filters from spec with Arranger data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,24 @@ npm run relay
...
+To run with Arranger components:
+1. Set local environment variables:
+  - $STORYBOOK_ARRANGER_API: localhost:3000
+  - $STORYBOOK_PROJECT_ID: search
+  - $REACT_APP_ARRANGER_API: /api/v0/flat-search
+  - $REACT_APP_PROJECT_ID: search
+2. Run ElasticSearch at localhost:9200
+3. Run `gen3-arranger` server (`npm start`)
+4. Populate ElasticSearch DB using `es_gen_data()` function from `gen3-arranger`
+5. Follow the [Arranger](https://github.com/overture-stack/arranger) setup steps - run the server and the dashboard.
```
Comment (frickjack) [inline README.md]: Update this:
```
cd Docker/Stacks
docker-compose -f esearch.yml up -d
export ESHOST=localhost:9200
source esearch/indexSetup.sh
es_indices
es_delete_all
es_setup_index
es_gen_data 0 20
```

File: data/parameters.js [hunk 2]
```diff
@@ -1384,69 +1384,45 @@ const params = {
...
+        },
+        study: {
+          chartType: 'count',
+          title: 'Studies',
+        },
+        file_type: {
+          chartType: 'count',
+          title: 'File Types',
+        },
+        ethnicity: {
```
Comment (frickjack) [inline data/parameters.js]: Patch things so the explorer page still works even if `arrangerConfig` is not defined for a particular project.  Have a default `config` with some default fields (project, study, case, race, whatever) that the code falls back to ...

File: src/components/tables/DataExplorerTable/index.jsx [hunk 3]
```diff
@@ -1,62 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import DataTable from '@arranger/components/dist/DataTable';
+import { Table } from '@arranger/components/dist/Arranger';
 import '@arranger/components/public/themeStyles/beagle/beagle.css';
-import fileDownload from 'js-file-download';
```
Comment (frickjack) [inline src/components/tables/DataExplorerTable/index.jsx]: Are we going to get rid of `beagle.css`?

File: data/parameters.js (hunk 2 above)
Comment (frickjack) [reply data/parameters.js]: Actually - it looks like you already handle this by defaulting `arrangerConfig` to `{}`  - awesome!
You can ignore the previous comment if that works ok ...

File: src/components/tables/DataExplorerTable/index.jsx (hunk 3 above)
Comment (abgeorge7) [reply src/components/tables/DataExplorerTable/index.jsx]: PXD-1325

File: src/components/ArrangerWrapper.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Arranger } from '@arranger/components/dist/Arranger';
+import { AggsState, AggsQuery } from '@arranger/components/dist/Aggs';
+
+class ArrangerWrapper extends React.Component {
```
Comment (frickjack) [inline src/components/ArrangerWrapper.jsx]: Documentation please - what is this, when and how do I use it ...?

**Pull request merged**